quick-start/ruby.md

```diff
@@ -63,7 +63,7 @@ on Rails application.
     source ~/.zshrc
     ```
 
-1.  Add TimescaleDB to Rails migration:
```

Comment:
In this case, I guess we keep it as it's the extension itself right?

Reply:
I think since we're emphasizing the cloud service, we're adding the service on the platform.  I know we're _technically_ adding the extension, but since the actual command is the same, it's less confusing I think if we use "Timescale" rather than referring to TimescaleDB.  Or alternatively, change it to "Add the TimescaleDB extension to your Rails migration"